src/index.js

```diff
@@ -29,3 +31,26 @@ httpServer.listen({ port }, () => {
     `Subscriptions ready at ws://localhost:${port}${server.subscriptionsPath}`
   )
 })
+
+//Sequelize usage test
+const dummy_user = {
+  firstName: "Ezequiel",
+  lastName: "Giachero",
+  username: "Eze123",
+  salt: "",
+  password: "pepito"
+}
+
+async function createShowDestroy(user){
```

Comment:
Some code-style issues. Make sure to set up and run a linter before pushing code.